website/versions.json

```diff
@@ -1,11 +1 @@
-[
-  "26.6",
-  "26.5",
-  "26.4",
-  "26.2",
-  "26.0",
-  "25.x",
-  "24.x",
-  "23.x",
-  "22.x"
-]
+["26.6", "26.5", "26.4", "26.2", "26.0", "25.x"]
```

Comment:
rather than deleting old versions I'd rather consolidate all 26.x into a single one (27 is coming soon at which point that'd happen regardless)